src/test/handlers/featureFlags.test.js

```diff
@@ -0,0 +1,59 @@
+const request = require('supertest');
+const config = require('../../config/config.js');
+const support = require('./_support.js');
+
+describe('GET /featureflags', () => {
+    describe('Failures:', () => {
+        test('should return 200 and false for transferVerification when config is set to "false"', async () => {
+            const confOverride = JSON.parse(JSON.stringify(config));
+            confOverride.featureFlags.transferVerification = false;
+
+            const db = support.createDb();
+            const server = support.createServer(db, undefined, undefined, confOverride);
```

Comment:
It is cleaner to put the repeating code in  beforeEach/afterEach blocks to avoid duplication